src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Default {
+    private double size;
+
+    public Circle(String color, double size) {
+        super(color);
+        this.size = size;
+    }
+
+    @Override
+    public double getArea() {
+        return size * size;
```

Comment:
The formula for the area of a circle is incorrect. It should be `Math.PI * size * size` to calculate the area using the radius.